Documentation/resources.adoc

```diff
@@ -74,6 +74,29 @@ Expose the user-defined Alertmanager web server within the cluster on the follow
 
 Expose the Alertmanager web server within the cluster on the following ports:
 * Port 9094 provides access to all the Alertmanager endpoints. Granting access requires binding a user to the `monitoring-alertmanager-view` role (for read-only operations) or `monitoring-alertmanager-edit` role in the `openshift-monitoring` project.
+```
+$ oc project openshift-monitoring
+$ oc create serviceaccount am-ro-client
+$ oc adm policy add-role-to-user monitoring-alertmanager-view \
+  --role-namespace=openshift-monitoring --rolebinding-name=am-ro-client \
+  --serviceaccount=am-ro-client
+
+$ TOKEN=$(oc create token am-ro-client)
+$ ROUTE=$(oc get route alertmanager-main -n openshift-monitoring -ojsonpath={.spec.host})
+$ curl -H "Authorization: Bearer $TOKEN" -k "https://$ROUTE/api/v2/alerts?filter=alertname=Watchdog"
+---
+
+$ oc project openshift-monitoring
+$ oc create serviceaccount am-rw-client
+$ oc adm policy add-role-to-user monitoring-alertmanager-edit \
+  --role-namespace=openshift-monitoring --rolebinding-name=am-rw-client \
+  --serviceaccount=am-rw-client
+
+$ TOKEN=$(oc create token am-rw-client)
+$ ROUTE=$(oc get route alertmanager-main -n openshift-monitoring -ojsonpath={.spec.host})
+$ curl -H "Authorization: Bearer $TOKEN" -k "https://$ROUTE/api/v2/status"
+---
+```
 * Port 9092 provides access to the Alertmanager endpoints restricted to a given project. Granting access requires binding a user to the `monitoring-rules-edit` cluster role or `monitoring-edit` cluster role in the project.
 * Port 9097 provides access to the `/metrics` endpoint only. This port is for internal use, and no other usage is guaranteed.
```

Comment:
Hello @machine424 if it is not too much trouble, would it be possible to add as part of this PR one small change? To this section ([openshift-monitoring/thanos-querier](https://github.com/openshift/cluster-monitoring-operator/pull/2443/files#diff-1849614964d69a5b648750198efb5529dc00bb92dd56cb882cc6495f199bda9dL130-L133)):

from:
* Port 9091 provides access to all the Thanos Querier endpoints. Granting access requires binding a user to the `cluster-monitoring-view` cluster role.

to:

* Port 9091 provides access to all the Thanos Querier endpoints. Granting access requires binding a user to the `cluster-monitoring-view` cluster role or `cluster-monitoring-metrics-api` role.

It was raised in these comments: https://github.com/openshift/openshift-docs/pull/80127#discussion_r1716908947

cc: @simonpasquier

Reply:
I’m afraid this PR may take some time before getting merged. If the change is urgent or blocking something else, I think it’s better to do it in another PR.

Reply:
I'll take care of this in another PR.

Reply:
Taking a look. this would require some changes in the utils, opened https://issues.redhat.com/browse/MON-4053